qa-core/src/internal-api/tests/users/userManagement.spec.ts

```diff
@@ -87,4 +87,20 @@ describe("Internal API - User Management & Permissions", () => {
     expect(changedUserInfoJson.builder?.global).toBeDefined()
     expect(changedUserInfoJson.builder?.global).toEqual(true)
   })
+
+  it("Set First and Last Name", async () => {
+    const body: Partial<User> = {
+      firstName: "newFirstName",
+      lastName: "newLastName",
+    }
+    await config.api.self.changeUserInfo(body)
```

Comment:
Can we do a follow up request to load the user and verify the changes have been applies?